_sass/catalog.scss

```diff
@@ -376,7 +376,7 @@ input[type="checkbox"] {
   width: 25%;
   padding: 2.5rem;
   // height: fit-content;
-  height: 100%;
+  height: 103.25%;
```

Comment:
The decimal percentage width might seem harmless now, but it could lead to issues down the line. For example, when adjusting the height of other areas, it could cause bugs in this section. This could create a cycle where we constantly have to come back and readjust. Try to wrap the whole modal and give it a height according to its content or some max height